HDPS/src/private/PluginManager.cpp

```diff
@@ -632,11 +632,16 @@ void PluginManager::fromVariantMap(const QVariantMap& variantMap)
         if (!_pluginFactories.contains(usedPlugin.toString()))
             missingPluginKinds << usedPlugin.toString();
 
-    if (!missingPluginKinds.isEmpty())
-        throw std::runtime_error(QString("One or more plugins are not available: %1").arg(missingPluginKinds.join(", ")).toLocal8Bit());
-
     if (variantMap.contains("LoadedAnalyses"))
     {
+        for (const auto& loadedAnalysis : variantMap["LoadedAnalyses"].toList())
+            missingPluginKinds.removeAll(loadedAnalysis.toMap()["Kind"].toString());
+
+        missingPluginKinds.squeeze();
+
+        if (!missingPluginKinds.isEmpty())
+            throw std::runtime_error(QString("One or more plugins are not available: %1").arg(missingPluginKinds.join(", ")).toLocal8Bit());
+
```

Comment:
If an analysis plugin was used in the saved session but not saved to the project (e.g. because the plugin is not serialized), we do not need to load it again. Hence, it is not missing in case it's not available.